classes/abstracts/ActionScheduler.php

```diff
@@ -297,9 +300,11 @@ protected static function is_class_migration( $class ) {
 	 */
 	protected static function is_class_cli( $class ) {
 		static $cli_segments = array(
-			'QueueRunner' => true,
-			'Command'     => true,
-			'ProgressBar' => true,
+			'QueueRunner'                          => true,
+			'Command'                              => true,
+			'ProgressBar'                          => true,
+			'ActionScheduler_WPCLI_Action_Command' => true,
+			'ActionScheduler_WPCLI_System_Command' => true,
```

Comment:
Do these need to be updated following the switch to namespacing?

Reply:
@barryhughes I absolutely hate this loading mechanism. May need your help to figure this out.

Reply:
Okay, think I got it.

Reply:
Thanks. Yep, it's a little tricky to navigate.